modules/tinymce/src/core/test/ts/browser/FormatterApplyTest.ts

```diff
@@ -497,15 +497,15 @@ describe('browser.tinymce.core.FormatterApplyTest', () => {
         fontWeight: 'bold'
       }
     });
-    editor.getBody().innerHTML = '<p>a<span id="id" style="font-weight:bold">1234</span>b</p>';
+    editor.getBody().innerHTML = '<p>a<span id="test-id" style="font-weight:bold">1234</span>b</p>';
```

Comment:
Changed the IDs (here and elsewhere) to avoid tripping "prevent DOM Clobbering" checks